data/projects/generic_bears.md

```diff
@@ -0,0 +1,28 @@
+The Project is about fixing issues with Generic Bears.
+
+Generic Bears have the potential to be awesome, and they have started to get
+their, but still require a some issues to be fixed. For eg: QuotesBear is not
+reliable, AnnotationBear needs to give a better more detailed API, 
+IndentationBear looks cool but is not usable yet. These bears can be found 
+[here](https://github.com/coala/coala-bears/tree/master/bears/general).
+
+The task is to fix as many issues with these bears and make them as usable
+as possible.
+
+### Milestones
+
+##### GSoC 2017 Coding Starts
```

Comment:
maybe sonthing like "the student has identified at least # important issues" or so?